packages/backend/src/handlers/actions/3Box/routes.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+import { asyncHandlerWrapper } from '../../../lib/apiHelpers';
+import { cache3BoxProfileHandler } from './cache3BoxProfile/handler';
+import { refresh3BoxCacheHandler } from './refresh3BoxCache/handler';
```

Comment:
naming is confusing. there's cacheProfile, updateCachedProfile, and refreshCache and its not clear which one does what since they're used interchangeably. Would be good to unify the naming a bit

Reply:
The names are now `updateSingleProfile` for the individual invalidation and `updateExpiredProfiles` for the process that checks for any profiles not updated in the last four days.